### PR TITLE
Em

### PR DESCRIPTION
**Pull request by Shubhambawner**
Team page updated 
now its more dynamic, componentised, and error tollerent.
EM added.

### CODE REVIEW

**Comment by Shubhambawner** [comment]
resolves #28

**Comment by Shubhambawner** [comment]
@AyushAgrawal25 @rajesh6161

**Comment by sahil9001** [comment]
> resolves #28 
> 
> partially resolves #27 

One PR should resolve one issue because when we have to revert we can't revert separately. @Shubhambawner  revert #27 changes and keep this PR as #28 only.

File: src/Components/Team/descriptions.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    MNG: [2,'Manager',],
+    EXC: [1,'Executive'],
+
+    none: '',
+    tech: 'Technical Team',
+    doc: 'Documentation Team',
+    pr: 'Public Relation and Marketing',
+    spons: 'Sponsorship Team',
+    design: 'Design Team',
+    em: 'Event Management',
```
Comment (sahil9001) [inline src/Components/Team/descriptions.js:18]: Add Video editing domain as 've'

**Review by sahil9001 — CHANGES_REQUESTED** [review]

**Comment by Shubhambawner** [comment]
> > resolves #28
> > partially resolves #27
> 
> One PR should resolve one issue because when we have to revert we can't revert separately. @Shubhambawner revert #27 changes and keep this PR as #28 only.

Oh yes, after some examination I found that issue 27 is independent. initially it was incorrectly linked.

File: src/Components/Team/descriptions.js (hunk 1 above)
Comment (Shubhambawner) [reply src/Components/Team/descriptions.js:18]: done

**Comment by sahil9001** [comment]
@AyushAgrawal25 I haven't tested these files but looks good to me, can you check it?

**Review by AyushAgrawal25 — APPROVED** [review]
LGTM 
I am merging the changes.

**Review by AyushAgrawal25 — CHANGES_REQUESTED** [review]
Change the base_url at axios.js to https://ecellbackend.tech/, currently it's causing merge issue.

**Pull request merged**